BookReader/BookReader.js

```diff
@@ -170,6 +170,10 @@ function BookReader() {
     // Experimental Controls (eg b/w)
     this.enableExperimentalControls = false;
 
+    // Base server used by some api calls
+    this.server = '';
+    this.searchInsideUrl = '/fulltext/new_inside.php';
```

Comment:
Note, I made this endpoint configurable because we are moving to "inside.php" soon, and having it configurable will make it easy to roll this out.